src/main/java/org/apache/commons/io/FileSystem.java

```diff
@@ -411,30 +424,26 @@ public int getMaxFileNameLength() {
     /**
      * Gets the maximum length for file paths (may include folders).
      *
-     * <p><strong>Note:</strong> This may include folder names as well as the file name.
-     * The unit is the same as {@link #getMaxFileNameLength()} and can be obtained
-     * from {@link #getLengthUnit()}.</p>
+     * <p>This may include folder names as well as the file name.</p>
```

Comment:
“May include folder names” sounds a bit unclear to me, is this the path length (including separators and roots) for folders and files or is it for path components?